sammo/throttler_test.py

```diff
@@ -1,20 +1,19 @@
-import pytest
-from pytest import approx
 import asyncio
+import threading
 import time
-from sammo.throttler import Throttler, AtMost
 
+import pytest
+from pytest import approx
 
-async def sleep(delay: float) -> None:
-    await asyncio.get_running_loop().run_in_executor(None, time.sleep, delay)
+from sammo.throttler import Throttler, AtMost
 
 
 async def simple_job(job_id, throttler, fail=False, delay=0):
     scheduled = time.perf_counter()
     job = await throttler.wait_in_line()
     run = time.perf_counter()
     if delay > 0:
-        await sleep(delay)
+        await asyncio.sleep(delay)
```

Comment:
@t-schn I think this preserves the functionality of the separate sleep method. The call to `time.sleep` seemed to be adding some non-determinism in the GitHub Actions runner environment, which is much noisier than our dev environments. These test failures only showed up here - everything worked fine for me on my dev machine.